doc/development/local_setup.md

```diff
@@ -0,0 +1,76 @@
+## Prerequisites
+
+Although the following installation instructions are for Mac OS X, similar alternate commands could be found for any Linux distribution
+
+### Installing [Golang](https://golang.org/) environment
+
+Install the latest version of Golang (at least `v1.12` is required). For Mac OS, you could use [Homebrew](https://brew.sh/):
+
+```sh
+brew install golang
+```
+
+For other OS, please check [Go installation documentation](https://golang.org/doc/install).
+
+Make sure to set your `$GOPATH` environment variable properly (conventionally, it points to `$HOME/go`).
+
+For your convenience, you can add the `bin` directory of the `$GOPATH` to your `$PATH`: `PATH=$PATH:$GOPATH/bin`, but it is not necessarily required.
+
+### [Golint](https://github.com/golang/lint)
+
+In order to perform linting on the Go source code, please install [Golint](https://github.com/golang/lint):
+
+```bash
+go get -u golang.org/x/lint/golint
+```
+
+### [Ginkgo](https://onsi.github.io/ginkgo/) and [Gomega](https://onsi.github.io/gomega/)
+
+In order to perform tests on the Go source code, please install [Ginkgo](https://onsi.github.io/ginkgo/) and [Gomega](http://onsi.github.io/gomega/). Please make yourself familiar with both frameworks and read their introductions after installation:
+
+```bash
+go get -u github.com/onsi/ginkgo/ginkgo
+go get -u github.com/onsi/gomega
+```
+
+### Installing `git`
+
+We use `git` as VCS which you would need to install.
+
+On Mac OS run
+
+```sh
+brew install git
+```
+
+### Installing `gcloud` SDK (Optional)
+
+In case you have to create a new release or a new hotfix, you have to push the resulting Docker image into a Docker registry. Currently, we are using the Google Container Registry (this could change in the future). Please follow the official [installation instructions from Google](https://cloud.google.com/sdk/downloads).
```

Comment:
```suggestion
In case you have to create a new release or a new hotfix, you have to push the resulting Docker image into a Docker registry. Currently, we use the Google Container Registry (this could change in the future). Please follow the official [installation instructions from Google](https://cloud.google.com/sdk/downloads).
```